packages/beacon-node/src/api/impl/lightclient/index.ts

```diff
@@ -2,38 +2,49 @@ import {routes} from "@lodestar/api";
 import {fromHexString} from "@chainsafe/ssz";
 import {SyncPeriod} from "@lodestar/types";
 import {MAX_REQUEST_LIGHT_CLIENT_UPDATES, MAX_REQUEST_LIGHT_CLIENT_COMMITTEE_HASHES} from "@lodestar/params";
+import {LightClientUpdate} from "@lodestar/types/altair";
 import {ApiModules} from "../types.js";
 
 // TODO: Import from lightclient/server package
 
-export function getLightclientApi({chain}: Pick<ApiModules, "chain">): routes.lightclient.Api {
+export function getLightclientApi({chain, config}: Pick<ApiModules, "chain" | "config">): routes.lightclient.Api {
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  const lightClientUpdatesWithVersion = (chunks: LightClientUpdate[]) => {
```

Comment:
What's the need to declare a function here? Just paste the logic in `getUpdates`

Reply:
@dapplion updated as suggested